executor/distsql_statementContextToFlags_test.go

```diff
@@ -0,0 +1,15 @@
+package executor
```

Comment:
Why use this file name?

Reply:
`distsql_test.go` use package `executor_test`, not package `executor`.
It's means we couldn't call function `statementContextToFlags` in test case, because it is private (not exported).

Reply:
Maybe we can refactor it and make it as a method of `StatementContext`. Anyway, the file name is not satisfied with our convention.